### ISSUE TITLE
Add a filter for non-zero findings in OpenScanHub dashboard

### ISSUE THREAD

**Issue by siteshwar** [packit/dashboard#496]
### Description

OpenScanHub [dashboard](https://dashboard.packit.dev/jobs/openscanhub) should have a filter for non-zero findings to make it easier to find packages that may be introducing new bugs. Currently, it displays large amount of tasks with zero findings.

### Benefit

It would make it easier for Packit and OpenScanHub developers to manually monitor tasks and report findings upstream until it is done automatically.

### Importance

Currently maintainers may be ignoring findings that may be introducing real bugs in the code.

### Workaround

- [ ] There is an existing workaround that can be used until this feature is implemented.

### Participation

- [ ] I am willing to submit a pull request for this issue. (Packit team is happy to help!)

**Comment by Venefilyn** [comment]
Currently the dashboard is heavily limited by current general API structure. I have a few issues opened over on the packit-service front to improve this, but the specific issue I'd say is the most important is moving to FastAPI or generally restructuring the View-Model situation -- both of which would take some time but would mean adding filtering, searching, and ordering would be a piece of cake.

- https://github.com/packit/packit-service/issues/2567
- https://github.com/packit/packit-service/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+author%3AVenefilyn

In short, not much we can do on the dashboard front that wouldn't be extremely API heavy or generally ugly with client-side filtering.
